MaxText/maxengine_server.py

```diff
@@ -0,0 +1,50 @@
+"""Runs a server with maxtext."""
+
+from typing import Sequence
+
+from absl import app
+from absl import flags
+import jax
+import os
+import sys
+import pyconfig # change the path to maxtext pyconfig
+
+import  maxengine_config
+from jetstream.core import server_lib
```

Comment:
JetStream need to be added in requirements.txt, I don't see it in there rn. Is that auto added through a different mechanism?

Reply:
needs to be added